server/src/internalClusterTest/java/org/opensearch/cluster/SimpleClusterStateIT.java

```diff
@@ -90,7 +95,19 @@
  * Checking simple filtering capabilities of the cluster state
  *
  */
-public class SimpleClusterStateIT extends OpenSearchIntegTestCase {
+public class SimpleClusterStateIT extends ParameterizedStaticSettingsOpenSearchIntegTestCase {
+
+    public SimpleClusterStateIT(Settings settings) {
+        super(settings);
+    }
+
+    @ParametersFactory
+    public static Collection<Object[]> parameters() {
+        return Arrays.asList(
+            new Object[] { Settings.builder().put(CLUSTER_REPLICATION_TYPE_SETTING.getKey(), ReplicationType.DOCUMENT).build() },
+            new Object[] { Settings.builder().put(CLUSTER_REPLICATION_TYPE_SETTING.getKey(), ReplicationType.SEGMENT).build() }
```

Comment:
I don't think we *always* need to run this with SR on/off.  Can we randomly select the replication strategy?